pkg/generated/BUILD

```diff
@@ -12,6 +12,9 @@ load(
 
 go_library(
     name = "go_default_library",
-    srcs = ["doc.go"],
+    srcs = [
+        "bindata.go",
```

Comment:
There are two reasons for compiling in:
1) Load time performance so that we don't slow down kubectl by loading lots of files.
2) Packaging, it's really good that right now you can ship `kubectl` and nothing else and it just works.  Translations should be no different.

(and hey `kubectl` is already > 70Mb)